keps/sig-node/2400-node-swap/README.md

```diff
@@ -343,7 +343,7 @@ type KubeletConfiguration struct {
 
 type MemorySwapConfiguration struct {
 	// Configure swap memory available to container workloads. May be one of
-	// "", "NoSwap": workloads cannot use swap
+	// "", "LimitedSwap": workload combined memory and swap usage cannot exceed pod memory limit
```

Comment:
Can we clarify that v1 and v2 behavior will diverge here?

Reply:
It won't diverge in the current implementation, as we're not disallowing swap usage on cgroupsv2. If we want to allow for a "noswap" option in the future, we can add that for v2 only.

Reply:
Yes, but we can clarify that this LimitedSwap option only applies to cgroups v1 and not v2.

Reply:
It applies to both right now.

Reply:
@mrunalp and I synced offline; I was unaware that cgroupsv2 doesn't support a combined memory+swap limit, and must configure them separately.

This behaviour difference is now addressed below!